### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by yichuandoris** [yichuandoris/MI-449-SS17-740-html-multiple-pages-T_njfg#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://mi-449-ss17-740-html-multiple-pages-t_njfg-yichuandoris.surge.sh/) and meets the following criteria:

- [x] None of the routes on the website end with `.html`
- [x] A homepage exists, where the purpose of the site is explained
- [x] A "list page" exists that lists your top 3 favorite of this thing (e.g. `/pokemon`) -- each list item should link to the corresponding page for that favorite, detailed below
- [x] For each of your favorites, a page exists to display more information about it (e.g. `/pokemon/bulbasaur`)
- [x] Every page includes links to the homepage and list page (unless either of those are the current page)

I am not sure if I should add more information about the those items. Please let me know if I should do so. And I will update it. Thank you.  -Doris

**Comment by chrisvfritz** [comment]
You don't have to add more information. 🙂  This looks really good and demonstrates that you can do the things we really care about for this lesson. The only room for improvement that I see is in the indentation. The code [inside the body here](https://github.com/yichuandoris/MI-449-SS17-740-html-multiple-pages-T_njfg/blob/master/list/index.html#L9-L20) isn't quite indented beyond the `body` element and there are some other irregularities too.

For example, in this code:

``` html
<a href="/">
Home
</a>
```

Home is inside the `a` element, but is not indented inside it, like this:

``` html
<a href="/">
  Home
</a>
```

Here's another example:

``` html
<p>
  See other Characters?<a href="/list">List
  </a>
</p>
```

Whenever an element covers multiple lines, I expect the opening and closing tags to be on their own line and the content indented inside it. For this would be acceptable:

``` html
<p>
  See other Characters?
  <a href="/list">
    List
  </a>
</p>
```

Or this, with the entire `a` element on its own line:

``` html
<p>
  See other Characters?
  <a href="/list">List</a>
</p>
```

Or even this:

``` html
<p>
  See other Characters?<a href="/list">List</a>
</p>
```

Let me know if you have questions about these ideas and I'm happy to help here, on Slack, or we could also do a video call with screen sharing if that would be easier. 😸

**Comment by yichuandoris** [comment]
Thanks Chris. I've made the indentation revisions and pushed my code to git-hub again (also updated the live site). Could you check it again? Thanks.

Also, I have a question about the Atom. Should we "save" each "index" page to make sure all changes in the related pages (such as the index in the other directories) are saved? Or we only need to save the main index (the home page)? Do you see what I mean? If not, I can try asking it another way. Thanks. - Doris

**Comment by chrisvfritz** [comment]
Looks much better! And by the way, I really liked your use of the `button` class to make those links prettier! ⭐️  There's only [one more block of code](https://github.com/yichuandoris/MI-449-SS17-740-html-multiple-pages-T_njfg/blob/master/list/lugi/index.html#L8-L10) where the indentation could still be better. 

Regarding Atom, I think I understand, but please correct me if I have not. 🙂  You have to individually save any page that you change in Atom, but for the ones that haven't been changed, then you don't need to do anything. Does that make sense?

**Comment by yichuandoris** [comment]
Yes. Thanks for answering that question. And you know what, I just found myself forgot to save the index for "lugi" page before I pushed the code. I think I just remembered to save it before I closed Atom, becuase when I opened Atom, the "missing error" was actually fixed. I hope everything is alright now. - Doris

**Comment by chrisvfritz** [comment]
Looks perfect! 🎉 🎈 :shipit:
